knowledge/external/horizon-api.md

```
@@ -0,0 +1,243 @@
+# Meta Horizon IAP API Reference
+
+> External reference for Meta Horizon Store in-app purchase APIs.
+> Source: https://developers.meta.com/horizon/documentation/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)